src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/AspNetCore6Plus/ResponseStreamWrapper.cs

```diff
@@ -36,18 +35,6 @@ public override void Flush()
             _baseStream.Flush();
         }
 
-        public override Task FlushAsync(CancellationToken cancellationToken)
-        {
-            if (!_isContentLengthSet && IsHtmlResponse())
-            {
-                _context.Response.Headers.ContentLength = null;
-                _isContentLengthSet = true;
-            }
```

Comment:
Is this code not necessary anymore?

Reply:
No, it was from the example code I originally was working with, that was for a slightly different purpose than what we're doing.